Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -52,87 +52,95 @@ directives.
-    few inserts/deletes), a higher value will impact performance without any
-    added benefits
-
-  * **commitIntervalMsec** (_optional_, default: `60000`; to disable use: `0`)<br/>
-    wait at least *count* milliseconds between committing view data store
-    changes and making documents visible to queries
-    for the case where there are a lot of inserts/updates, a lower value will
-    cause the view not to account for them, (unlit commit), and memory usage
-    would continue to grow
-    for the case where there are a few inserts/updates, a higher value will
-    impact performance and waste disk space for each commit call without any
-    added benefits
-
-  * **consolidate** (_optional_, default: `none`)<br/>
-    a per-policy mapping of thresholds in the range `[0.0, 1.0]` to determine data
-    store segment merge candidates, if specified then only the listed policies
-    are used, keys are any of:
-
-    * **bytes** (_optional_, for default values use an empty object: `{}`)
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > segment_bytes / (all_segment_bytes / #segments)`
-
-    * **bytes_accum** (_optional_, for default values use: `{}`)<br/>
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > (segment_bytes + sum_of_merge_candidate_segment_bytes) / all_segment_bytes`
-
-    * **count** (_optional_, for default values use: `{}`)
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > segment_docs{valid} / (all_segment_docs{valid} / #segments)`
-
-    * fill: (optional)
-      if specified, use empty object for default values, i.e. `{}`
-
-      * **segmentThreshold** (_optional_, default: `300`; to disable use: `0`)<br/>
-        apply consolidation policy IFF {segmentThreshold} >= #segments
-
-      * **threshold** (_optional_, default: `0.85`)<br/>
-        consolidate `IFF {threshold} > #segment_docs{valid} / (#segment_docs{valid} + #segment_docs{removed})`
+  * link removal - JSON keyword *null* (i.e. nullify a link if present)<br/>
+* any of the directives from the section [View properties](#view-properties)
+
+## View properties
+
+* **cleanupIntervalStep** (_optional_; default: `10`; to disable use: `0`)<br/>
+  wait at least this many commits between removing unused files in the
+  ArangoSearch data directory
+  for the case where the consolidation policies merge segments often (i.e. a
```

Comment:
Don't know what consolidation means in this case. Either explain it here or better, refer to a place where it is explained.

Reply:
Added paragraph with explanation.